plugin-server/src/main/ingestion-queues/session-recording/session-recordings-blob-consumer.ts

```diff
@@ -280,8 +280,10 @@ export class SessionRecordingBlobIngester {
             if (err.code === CODES.ERRORS.ERR__ASSIGN_PARTITIONS) {
                 /**
                  * The assign_partitions indicates that the consumer group has new assignments.
-                 * We don't need to do anything, but it is useful to log for debugging.
+                 * We explicitly ensire the offset manager is not treating these partitions as revoked
```

Comment:
Finally I'm the one finding typos 😅 

```suggestion
                 * We explicitly ensure the offset manager is not treating these partitions as revoked
```

Reply:
No, "ensire" is archaic english for, erm... making something the parent of an animal as an, erm, animal, and...

![homer-hedge](https://github.com/PostHog/posthog/assets/984817/6e57db0d-d09a-4fe9-a76c-39d8de03f22e)